pyproject.toml

```diff
@@ -72,7 +72,7 @@ version-file = "src/fixit/__version__.py"
 features = ["dev", "pretty"]
 
 [tool.hatch.envs.default.scripts]
-test = "python -m fixit.tests"
+test = "python3 -m fixit.tests"
```

Comment:
Why was this change needed?  Hatch environments should be controlled enough that just `python` will always work.

Reply:
I think this being "python3" is what's causing the CI failures on windows. My guess is that the virtualenv doesn't have a python3.exe in it, and so it's finding a random non-venv runtime, and therefore can't find/import fixit.